src/validateEmail.test.js

```diff
@@ -17,4 +17,138 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+  
+  it('should return "false" non-english letters', () => {
+   expect(validateEmail('bąk@doom.com.'))
```

Comment:
The email 'bąk@doom.com.' is considered invalid because it ends with a dot. The expectation should be `toBeFalsy()` instead of `toBe(false)`.

Reply:
it returns true with dot at end, but it should-t